specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -609,6 +1322,26 @@ components:
           format: "date"
         indicatieOudeOnroerendeZaakBetrokken:
           type: "boolean"
+        isGebaseerdOpStukdeelIdentificatie:
+          type: "string"
+          description: "De identificatie van het stukdeel waarop dit erfpachtcanon is gebaseerd"
+        isVermeldInStukdeelIdentificaties:
+          type: "array"
+          description: "De identificaties van de stukdelen waarin dit erfpachtcanon is vermeld"
+          items:
+            type: "string"
+    Filiatie:
+      type: "object"
+      properties:
+        aard:
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/master/specificatie/BRK-Bevragen/openapi.yaml#/components/schemas/Waardelijst"
+        overgangsgrootte:
+          description: "De oppervlakte van het gedeelte van de oude Onroerende zaak dat opgegaan is in de nieuwe Onroerende zaak. De overgangsgrootte heeft uitsluitend betrekking op Onroerende zaak van het soort Perceel."
+          type: "integer"
+          example: 220
+        kadastraalOnroerendeZaakIdentificaties:
+          description: "De identificatie(s) van de Kadastraal Onroerende Za(a)k(en)."
+          type: "string"
```

Comment:
als het er meerdere kunnen zijn moet het een array zijn. Maar filiatie zit al altijd in array, dus binnen een filiatie is het wellicht 1 kadastraal object?